### ISSUE TITLE
ACCESSIBILITY - Add label to mailing list subscribe feature in footer

### ISSUE THREAD

**Issue by Roenok** [girldevelopit/gdi-website#401]
**Description**
In the footer of the site, we have a tiny form that lets users subscribe to our mailing list. The text input for email address doesn't have a "programmatically determined name," a fancy name for a label. Because of this, users using assistive technology like a screen reader or voice-control software might not be able to select the box or join the mailing list.

**Ask**
Generally, forms use `<label>` tags to label text input fields. However, the design of that form makes using a label impractical. Here, it might make sense to add a title attribute to the field to give it a name.

**To Do**
- [ ] Add a `<label>` tag to the field OR a title attribute, with a sensible value like "Email address"
- [ ] Test the code with a screen reader (@roenok can help you test)
- [ ] Celebrate!

**Additional Info**
See [W3C – H65: Using the title attribute to identify form controls when the label element cannot be used](http://www.w3.org/TR/WCAG20-TECHS/H65.html) for more guidance on how to use the title attribute
